### ISSUE TITLE
Lister modal window and Add New issues in 2.6.14

### ISSUE THREAD

**Issue by thetuningspoon** [ryancramerdesign/ProcessWire#1363]
Okay, I must be getting good at breaking things. There seems to be an issue with Lister's Add New button in 2.6.14. When I click on the Add New button on the bottom of the screen, I briefly see the modal window but then the entire page seems to be redirected to the new page (still in modal=1 mode, but no longer in a separate iframe). The really weird thing is that the Add New button on the top of the screen still works fine.

The other issue is that deleting a page from within a modal window appears to refresh the lister (the row blinks), yet the page does not actually disappear from the list until a manual refresh using Lister's refresh button.

Also, the modal window also does not stay open when a required field is left empty (although the modal does refresh briefly before closing). Not sure what the intended behavior is with that, though.

**Comment by ryancramerdesign** [comment]
@thetuningspoon thanks, I was able to duplicate the lower "add new" button issue and have fixed it in ListerPro, so that fix will be in the next released version. The core Lister doesn't use/support the modal window option.

As an optimization I recently updated Lister/ListerPro to refresh only a single row rather than all the rows, after a page modification occurs. This makes things faster. The down side is that the page will continue to appear there (even if you trashed it) until all the rows are refreshed (whether manually or by clicking on a pagination, etc.). So that part is a minor compromise, but I think the overall benefits make it worth keeping. 

I can't duplicate the last issue you mentioned. When I leave a required field empty, the modal window continues to stay open, after I click "save" or "save and publish". Your issue report is from about a month ago, so perhaps this is something that I fixed during that time. I do recall this was an issue at one point, and then fixed, but let me know if you continue to experience it.
